test/benchmark/test/only.bench.ts

```diff
@@ -0,0 +1,68 @@
+import assert from 'node:assert'
```

Comment:
this file is not running by `test` script

and even then, it will never fail, because it just checks json